docs/DEVGUIDE.md

```diff
@@ -18,6 +28,22 @@ layout:
     ├── pkg            # Contains all non-"main" Go packages
     └── vendor         # Glide-managed dependencies (untracked)
 
+## Working on Issues
+
+Github does not allow non-maintainers to assign, or be assigned to, issues.
```

Comment:
Does this mean we should ignore or stop making use all-together of the Assignee in github?